src/pages/explore-quiz/ExploreQuiz.tsx

```diff
@@ -1,44 +1,49 @@
 import "./exploreQuiz.css";
 import { Footer, Navbar, QuizCard } from "../../components";
 import { useTheme } from "../../hooks";
+import { useEffect, useState } from "react";
+import { useQuiz } from "../../context";
+import { DocumentData } from "firebase/firestore";
+import { useParams } from "react-router-dom";
 
 const ExploreQuiz = () => {
 const { currentTheme } = useTheme();
-const quizData = [
-{
-quizName: "Know Your Nutrients",
-quizDescription: "Test your knowledge of nutrition.",
-quizImage:
-"/assets/nutrition.jpeg",
-quizStatus: "available",
-},
-{
-quizName: "fitness Quiz",
-quizDescription: "Coming Soon...",
-quizImage:
-"/assets/nutrition.jpeg",
-quizStatus: "not-available",
-},
-];
-return (
-<div className={`explore-quiz-container ${currentTheme==="dark" ? "dark" : "light" }`}>
-    <Navbar />
-    <main className="quiz-display-container">
-        {
-            quizData.map((quiz, index) => {
-                return <QuizCard key = {
-                    index
-                }
-                quiz = {
-                    quiz
-                }
-                />; 
-            })
-        } 
+const [quizData, setQuizData] = useState<DocumentData>();
+
+    const {
+    quizState: { quizzes },
+    } = useQuiz();
+
+    const { quizId } = useParams();
+
+    useEffect(() => {
+    setQuizData(quizzes.filter((quiz: DocumentData) => quiz.quizCategory === quizId));
+    }, [quizId, quizzes]);
+
+    console.log(quizId, "hello");
+    console.log(quizzes);
```

Comment:
You can remove the logs from here as well.

Reply:
Removed it, thanks1